components/schematics/src/fixtures/barista-components-package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@dynatrace/barista-components",
+  "version": "5.0.0",
+  "peerDependencies": {
+    "@angular/animations": "^8.2.12",
+    "@angular/cdk": "^8.2.12",
+    "@angular/core": "^8.2.12",
+    "@angular/common": "^8.2.12",
```

Comment:
NIT: the dependency for @angular/forms is missing in the fixture

Reply:
it is a fixture no need for it could be @bla/foo as well